tests/testStore.js

```diff
@@ -1,26 +0,0 @@
-import alt from '../src/alt/alt'
```

Comment:
That warning is from React 0.14, we shouldn't be checking the props of a
component, I can fix those tests up once you think the rest is good

On Thu, Oct 8, 2015 at 2:27 PM Ramana Venkata <notifications@github.com>
wrote:

> In tests/testStore.js
> <https://github.com/Legitcode/tests/pull/15#discussion_r41571396>:
>
> > @@ -1,26 +0,0 @@
> > -import alt from '../src/alt/alt'
>
> Didn't read anything regarding this in release notes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Legitcode/tests/pull/15/files#r41571396>.
>